uw_sws/enrollment.py

```diff
@@ -101,18 +102,23 @@ def _json_to_enrollment(json_data, term):
     enrollment.regid = json_data['RegID']
     enrollment.class_level = json_data['ClassLevel']
     enrollment.is_honors = json_data['HonorsProgram']
+
+    value = json_data['PendingMajorChange']
+    enrollment.has_pending_major_change = value
+
     enrollment.is_enroll_src_pce = is_src_location_pce(json_data,
                                                        ENROLLMENT_SOURCE_PCE)
-    enrollment.off_term_sections = {}
-    # dictionary {section_label: OffTermSectionReference}
-    if enrollment.is_enroll_src_pce:
-        if json_data.get('Registrations') is not None and\
-           len(json_data['Registrations']) > 0:
-            for registration in json_data['Registrations']:
-                if has_start_end_dates(registration):
-                    ot_section = _json_to_off_term_section(registration, term)
-                    key = ot_section.section_ref.section_label()
-                    enrollment.off_term_sections[key] = ot_section
+    enrollment.unf_pce_courses = {}
+    # dictionary {section_label: UnfinishedPceCourse}
+    if json_data.get('Registrations') is not None and\
```

Comment:
This is pretty dense code.  I think it would improve readability if the if statements that are currently over 2 lines were moved into a method.   The registration checks can probably be removed entirely: https://gist.github.com/vegitron/6a392622a63deae7c5ea94d60d017905